xbuild/src/lib.rs

```diff
@@ -354,6 +354,9 @@ pub struct BuildArgs {
     build_target: BuildTargetArgs,
     #[clap(flatten)]
     cargo: CargoArgs,
+    /// Path to a xbuild `manifest.yaml` to use for this build.
```

Comment:
Let's explain what the default case does (current package, either based on `$PWD` or `-p your-crate-name`, then the `manifest.yaml` in its root next to `Cargo.toml`):

```suggestion
    /// Path to an xbuild manifest to use for this build, instead of the default `manifest.yaml` in the crate root of the currently selected package.
```